main.css

```diff
@@ -200,3 +204,15 @@ footer {
   background-color: #1d1283;
   border: none;
 }
+
+#remainingDays {
+  /* all:unset; */
+  float: left;
+  margin-block: -.5em .75em;
+  padding-left: .5em;
+  font-size: 14px;
```

Comment:
If possible can we use em her for consistency

Reply:
I did use em